ddp-workspace/projects/ddp-prion/src/assets/i18n/en.json

```diff
@@ -155,7 +155,8 @@
       "BackToTop": "Back to top",
       "ContactUs": "Contact us:",
       "ContactMailTo": "mailto:info@prionregistry.org",
-      "ContactEmail": "info@prionregistry.org"
+      "ContactEmail": "info@prionregistry.org",
+      "PrivacyPolicy": "Privacy Policy"
```

Comment:
For any changes in en.json, make similar changes in he.json, es.json, and zh.json.  You can just put the same value as in en.json but put "(he)", "(es)", or "(zh)" in front.  This makes it easier to make sure things will internationalize correctly.

Reply:
Also, any header or footer changes need to be made in combination with changes to the Auth0 page, which involves code in ddp-study-server and making changes to the Auth0 Universal Login.  It's probably easier for me to do those.

Reply:
Similar changes in he.json, es.json, and zh.json are made. 
Removed prion footer changes from this branch. So it will be convenient for you update Auth0 along with your issue.